docs/pages/push-notifications/sending-notifications-custom.mdx

```diff
@@ -1,37 +1,37 @@
 ---
-title: Sending Notifications with APNs & FCM
+title: Send notifications with FCM & APNs
 hideFromSearch: true
 ---
 
-Communicating directly with APNs and FCM is much more complicated than sending notifications through [Expo's push notification service](sending-notifications.mdx), so you should only use this feature if you're prepared to undertake that complexity. Here are a few things you'll have to handle yourself if you choose to write your own server for FCM and APNs:
+Communicating directly with FCM and APNs is much more complicated than sending notifications through [Expo's push notification service]/push-notifications/sending-notifications/). You should only use this feature if you're prepared to undertake that complexity. Here are a few things you'll have to handle yourself if you choose to write your server for FCM and APNs:
 
-- Differentiating between native iOS & Android device tokens on your backend
+- Differentiating between native Android & iOS device tokens on your backend
 - Twice the amount of backend code to write and maintain (code for communicating with FCM, and then code for communicating with APNs)
 - Fetching responses from FCM and APNs to check if your notification went through, error handling, credentials management
 
-That being said, sometimes you need finer-grained control over your notifications, so communicating directly with APNs and FCM is necessary. The Expo platform does not lock you into using Expo's application services, and the `expo-notifications` API is push-service agnostic (you can use it with any push notification service).
+Sometimes you need finer-grained control over your notifications, so communicating directly with FCM and APNs is necessary. The Expo platform does not lock you into using Expo's application services, and the `expo-notifications` API is push-service agnostic.
```

Comment:
```suggestion
You may need finer-grained control over your notifications, so communicating directly with FCM and APNs may be necessary. The Expo platform does not lock you into using Expo's application services, and the `expo-notifications` API is push-service agnostic.
```

In my opinion, everything above this paragraph is not needed. I keep on reading (over and over) how Expo's push notification service is easy and removes complexity. That seems relevant for an overview page but I think we can remove that from pages like this one.